src/capi.zig

```diff
@@ -50,10 +51,16 @@ export fn compress(
 ) i32 {
     const uncompressed_values = uncompressed_values_array.data[0..uncompressed_values_array.len];
 
+    if (configuration.method > tersets.getMaxMethodIndex()) {
+        // This is equivalent to returning Error.UnknownMethod.
```

Comment:
Maybe make the if statement a one liner and move the comment above the if expression for consistency with the other checks. In this case the comment would probably be more clear if changed to something like "Returning 1 is equivalent to....".

Reply:
Fixed.